### ISSUE TITLE
Geometry Range in vector level for more precise lines

### ISSUE THREAD

**Issue by F-Burning** [opentoonz/opentoonz#4318]
**Is your feature request related to a problem? Please describe**
It is difficult to make precise lines with the brush using the Range function.

**Describe the solution you'd like**
The possibility of using this option with the geometry tool (rectangle, arc, multi-arc, etc.)

I know that they can be used in the level strip, but this way of working seems much more comfortable to me.

**Comment by F-Burning** [comment]
> You mean adding the frame range option to the geometry tool? I could see that being really helpful.

Yes, the truth is that I thought that option was also in the geometry tool.

**Comment by RodneyBaker** [comment]
This falls into the realm of autoinbetweening and curve animation.
REF:  #1878 

There are a few things that can get us closer to this.
One would be to port the update that allows vector autoinbetweening to be done via the timeline/xsheet.
REF:  https://github.com/tahoma2d/tahoma2d/pull/1428

A Frame Range feature for Geometry Tool or vector brush would need to tap into that inbetweening process.
